frescobaldi_app/job/lilypond.py

```diff
@@ -173,8 +173,18 @@ def configure_command(self):
         # just before starting the process.
         if sys.platform.startswith('darwin'):
             import macosx
-            if macosx.inside_app_bundle() and self.lilypond_info.frommacports():
-                self.environment['PATH'] = self.lilypond_info.bindir()
+            if macosx.inside_app_bundle():
+                bindir = self.lilypond_info.bindir()
+                if self.lilypond_info.frommacports():
+                    self.environment['PATH'] = bindir
+                else:
```

Comment:
MacPorts and Homebrew are not the only two possibilities: there is also the app bundle we provide. Please add a `fromhomebrew` method to `lilypondinfo.py` and use it here for an `elif`.

Reply:
Maybe it would be better to remove `frommacports` instead of adding a `fromhomebrew`. Adding the path from `lilypond` as a search path for other executables like `gs` seams to be a general solution which is not specific to MacPorts or Homebrew.